rebar.config

```
@@ -24,7 +24,7 @@
 %% == Dependencies ==
 
 {deps, [
-  {worker_pool, "3.1.0"},
+  {worker_pool, "5.1.0"},
```

```suggestion
  {worker_pool, "6.0.0"},
```

Latest worker pool is 6.0.0 now: https://hex.pm/packages/worker_pool/6.0.0